lib/inspec/resources/mysql_session.rb

```diff
@@ -16,19 +16,24 @@ class MysqlSession < Inspec.resource(1)
       end
     EXAMPLE
 
-    def initialize(user = nil, pass = nil, host = "localhost", port = nil, socket = nil)
+    def initialize(user = nil, pass = nil, host = "localhost", port = nil, socket = nil, sensitive: false)
```

Comment:
Hello! 👋  Thanks for the PR!

This is an interesting proposition. If we were to extend the functionality that we enjoy from the `command` resource to the `mysql_session` resource, would it be better to keep the DSL the same?

i.e. Instead of adding a new `sensitive: true` option, should we just make it the same as the `command` resource with `redact_regex`? ( Looking at the docs here: https://www.inspec.io/docs/reference/resources/command/ )

So maybe your code could look like `sql = mysql_session('my_user','password','db.example.com',  redact_regex: /(-p ).*( -d)/)`

I do think longer term it might be nice to support both a `redact_regex` (which gives the user lots of redacting power) as well as something to simply blocklist showing passwords, on both `command` and `mysql_session` and potentially other resources.

Side note: I had a look at your CI run/specs and I'm not sure if it actually is having the output you expected? https://buildkite.com/chef-oss/inspec-inspec-master-verify/builds/2000#5ef65086-d1db-43de-9ab9-d6deabc86869

Thanks again so much for sending this in, really appreciate it, and look forward to hearing from you! 🙏

Reply:
I don't foresee anything sensitive other than password for the time being. So maybe it makes sense to just use a proper regex to do that and don't let the user mess with it. Another option was to use the syntax below, but sadlyI don't know much about how to get there:
```
describe mysql_session('user','password','host'), :sensitive
```
For the time being, your proposal is very straightforward. Maybe it makes sense at first let the `redact_regex` be the initial available, and then we an think for an additional syntax that can be used like the `:sensitive` here.